UIKit_HM/UIKit_HM/ViewControllers/SettingsViewController.swift

```diff
@@ -0,0 +1,123 @@
+//  SettingsViewController.swift
+//  UIKit_HM
+//  Created by Мажит Закиров on 05.07.2021.
+
+import UIKit
+
+final class SettingsViewController: UIViewController {
+
+    // MARK: - Visial Companents
+
+    private let colorLabel = UILabel()
+    private let colorPicker = UIPickerView()
+    private let linesLabel = UILabel()
+    private let linesPicker = UIPickerView()
+
+    // MARK: - Life Circle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpView()
+    }
+
+    // MARK: - Delegate
+
+    var delegate: SettingsProtocol?
```

Comment:
```suggestion
    weak var delegate: SettingsProtocol?
```
грубо, грубо брат, утечка это плохо